sys/include/net/gcoap.h

```diff
@@ -304,7 +316,11 @@ extern "C" {
  *
  * Set to 0 to disable timeout.
  */
-#define GCOAP_NON_TIMEOUT       (5000000U)
+#ifndef MODULE_GNRC_DTLS
+# define GCOAP_NON_TIMEOUT    (5000000U)
```

Comment:
no space between `#` and `define` : you have this issue on other places as well. There's not indent with preprocessor directives